diagnostics/tropical_pacific_sea_level/dynamical_balance2.py

```diff
@@ -0,0 +1,268 @@
+import numpy as np
+
+
+####
+# Input variable are all xr.DataArray
+####
+
+####
+# constant 
+r_earth = 6.371*1E8         # cm
+
+def curl_var(da_uo,da_vo,x_name='lon',y_name='lat'):
+    """
+    Calculate the curl of the vector field 
+    
+    
+    Make sure 
+    
+    1. two Arrays are already regrid to the tracer points
```

Comment:
I didn't see where the POD is doing this regridding.

Reply:
Right, this is for original testing model output that does not have output on the tracer points. However, I found the model output provided in the CMIP6 OMIP archive seems to be on the same grid point which does not require regrid.